docs/src/main/paradox/healthchecks.md

```diff
@@ -0,0 +1,66 @@
+# Heath checks
+
+Akka Management supports two kinds of health checks:
+
+* Readiness checks: should the application receive external traffic, for example waiting for the following to complete
+    * Joining a Cluster
+    * Establishing a connection to a database or queuing system
+* Liveness checks: should the application be left running 
+
+Readiness checks can be used to decide if a load balancer should route traffic where
+as liveness checks can be used in environments which can restart a hung process.
+
+This matches [Kubernetes Health checks](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/).
```

Comment:
Should we also link to Colin Breck's advice? https://blog.colinbreck.com/kubernetes-liveness-and-readiness-probes-how-to-avoid-shooting-yourself-in-the-foot/

Reply:
Good idea